rocky/tools/view_helpers.py

```diff
@@ -153,3 +163,25 @@ def build_breadcrumbs(self):
                 "text": _("Objects"),
             }
         ]
+
+
+def schedule_task(request: HttpRequest, queue_name: str, item: QueuePrioritizedItem) -> None:
+    try:
+        client.push_task(queue_name, item)
+    except TooManyRequestsError as error:
```

Comment:
You can catch more than one type of error and use the same logic to handle them:
try:
  bla()
except (ConflictError, BadRequestError) as error:
  messages.error(request, error)